content/articles/2019_05_16_meetup.md

```diff
@@ -0,0 +1,30 @@
+Title: San Antonio Devs Meetup - Q2 2019
+Date: 02-12-2019
+Tags: meetup, events
+Slug: q2-2019-meetup
+Authors: Jon Roberts
+Summary: Information about the San Antonio Devs Q2 2019 meetup
+Event-start: 2019-05-16 6:30
+Event-duration: 150m
+Event-location: Luanch SA, 600 Soledad St. San Antonio, TX 78205
+
+We're excited to see you at our Q2 Meetup!  Please RSVP at the link Below.
+
+<https://www.eventbrite.com/o/san-antonio-software-developers-group-sadev-17973142435>
+
+Light refreshments will be available.
+
+
+Schedule:
+
+7pm        Welcome
+
+7:15       Jon Roberts - "Cloud-Native" Software Design and Architecture
+
+7:40       Break and Open Space event planning
```

Comment:
What does this mean?

Reply:
bigjonroberts   [< 1 minute ago]
to get input and ideas for topics to pursue for upcoming meetups